pl17_ex2/nos_tree.py

```diff
@@ -80,16 +98,16 @@ def nos_tree(S, s):
     print
     view_tree(tree)
 
```

Comment:
Sorry about the section bellow. I did some auto indentation, and forgot to revert that section